roles/helm_module/library/helm_cli.py

```diff
@@ -0,0 +1,190 @@
+import os
+import json
+
+from contextlib import contextmanager
+from tempfile import NamedTemporaryFile
+
+from ansible.module_utils.basic import *  # noqa
+
+
+class Helm(AnsibleModule):
+
+    def __init__(self, *args, **kwargs):
+        AnsibleModule.__init__(
+            self,
+            argument_spec=dict(
+                chart=dict(type='dict'),
+                release=dict(type='str', aliases=['name']),
+                namespace=dict(type='str'),
+                binary=dict(type='str'),
+                state=dict(default='present', choice=['present', 'absent']),
+                values_list=dict(type='list'),
+            )
+        )
+        self._helm_bin = None
+        self.delete_temp_file = True
+
+    def get_chart(self):
+        chart = self.params.get('chart')
+        if chart is None:
+            raise HelmError("'chart' is a mandatory argument for state=present")
+        if 'name' not in chart:
+            raise HelmError("'chart' argument required a 'name' key for state=present")
+        return chart
+
+    @property
+    def helm_bin(self):
+        if self._helm_bin is None:
+            binary = self.params.get('binary')
+            if binary and os.path.isfile(binary):
+                self._helm_bin = binary
+            else:
+                self._helm_bin = self.get_bin_path(
+                    'helm',
+                    required=True,
+                    opt_dirs=[binary] if binary else None)
+        return self._helm_bin
+
+    def get_release_info(self, release):
+        def failure_detect(rc, out, err):
+            return not(rc == 0 or
+                       'release: "{}" not found'.format(release) in err)
+
+        rc, out, err = self._run_helm(['get', release],
+                                      failed_when=failure_detect)
+        if rc != 0:
+            return
+        release_info = self._parse_helm_output(out)
+        return release_info
+
+    def install(self):
+        chart = self.get_chart()
+        release = self.params.get('release')
+        if release is None:
```

Comment:
There's no way to implement actual `present` without a given release name. Hence, I think (as said above) the actual implementation here is `latest`, not `present`, and if `present` were implemented it'd require `release` to be set.